src/components/button/button.tsx

```diff
@@ -67,16 +67,17 @@ const StyledButton = styled.button(
 			color: ${variant === 'pure'
 				? theme.pure
 				: isSecondaryDefault
-					? theme.button.default
-					: '#FFF'};
+				? theme.button.default
+				: '#FFF'};
 			background-color: ${disabled
 				? theme.button.disabled
 				: isPrimary
-					? theme.button.default
-					: !isColorDefault
-						? theme.button?.[setColor]
-						: 'transparent'};
-			border: 1px solid ${!isSecondary ? 'transparent' : isColorDefault ? theme.button.default : '#FFF'};
+				? theme.button.default
+				: !isColorDefault
+				? theme.button?.[setColor]
+				: 'transparent'};
+			border: 1px solid
+				${!isSecondary ? 'transparent' : isColorDefault ? theme.button.default : '#FFF'};
 			border-radius: ${pxToRem(6)};
 			transition: all 0.2s ease-in-out;
 			margin: ${isSecondaryDefault && '1px'};
```

Comment:
this should be solved differently because if it's not `secondaryDefault` it will be an emtpy style prop